src/yarpview/src/main.cpp

```diff
@@ -115,6 +139,19 @@ int main(int argc, char *argv[])
         window->setFlags(Qt::WindowStaysOnTopHint);
     }
 
+    PortConnector            connector;
+    yarp::os::Value          val;
+    yarp::os::ResourceFinder opt;
+    QObject::connect(topLevel, SIGNAL(connectToPort(QString, QString)), &connector, SLOT(connectPort(QString,QString)));
+
+    if (opt.configure(argc, argv))
+    {
+        if(opt.check("PortName")) connector.portName = opt.find("PortName").asString();
```

Comment:
Usually the name of the parameter starts with lower case, so i'd prefer `portName`

Reply:
it's not a parameter name we can change.. it is not a new parameter. it was already checked and used inside. i ve just pre-checked it here. There is so much space for refactoring here, expecially for the parameters parsing process but for the moment i made the less invasive change. leaving the refactoring for the future (it was not the pourpose of this pr)